build.gradle

```diff
@@ -1,6 +1,8 @@
-import java.nio.file.Files
```

Comment:
[build.gradle](build.gradle) becomes too heavy and needs to be reogranized. We may extract some part of it into `buildSrc` folder as described [here](https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#sec:build_sources)

Reply:
Moved part of this file to `distrib.gradle`